javagen/src/main/java/com/azure/autorest/mapper/ProxyMethodMapper.java

```diff
@@ -284,9 +286,31 @@ public Map<Request, ProxyMethod> map(Operation operation) {
             }
 
             ProxyMethod proxyMethod = builder.build();
+            proxyMethods.add(proxyMethod);
+
+            ProxyMethodParameter fluxByteBufferParam = parameters.stream()
+                    .filter(parameter -> parameter.getClientType() == GenericType.FluxByteBuffer)
+                    .findFirst()
+                    .orElse(null);
+
+            if (fluxByteBufferParam != null) {
+                List<ProxyMethodParameter> proxyMethodParameters = new ArrayList<>(parameters);
+                int i = parameters.indexOf(fluxByteBufferParam);
+                proxyMethodParameters.remove(i);
+
+                ProxyMethodParameter binaryDataParam = fluxByteBufferParam.toNewBuilder()
+                        .wireType(ClassType.BinaryData)
+                        .rawType(ClassType.BinaryData)
+                        .clientType(ClassType.BinaryData)
+                        .build();
+
+                proxyMethodParameters.add(i, binaryDataParam);
+                builder.parameters(proxyMethodParameters);
+                proxyMethods.add(builder.build());
+            }
```

Comment:
I will check behavior of fluentgen after merge. I assume if I want to disable for control-plane, I only need to condition on this?